lms/djangoapps/discussion/tests/test_views.py

```diff
@@ -76,6 +77,10 @@ def setUp(self):
             self.client = Client()
             assert_true(self.client.login(username=uname, password=password))
 
+        config = ForumsConfig.current()
+        config.enabled = True
+        config.save()
```

Comment:
(For archeological diggers, Cale pointed out to me that `super(..)` calls in Python are not strictly hierarchical and so by all mixed in and derived classes calling `super(...)` in overridden methods, all mixins could participate in, for example, running their own `setUp()`, which is what I did here.)